Training/PythonWebServer/webserver.py

```diff
@@ -2,20 +2,45 @@
 
 from http.server import SimpleHTTPRequestHandler, HTTPServer
 import sys
+import psutil
+from datetime import datetime
+import pytz
 
 class SoarWebServer(SimpleHTTPRequestHandler):
 
     def do_POST(self):
-        self.wfile.write("Send your POST response here!")
+        #Send your POST response here!
+        s0 = str(datetime.now(tz=pytz.timezone('America/Edmonton')).strftime('%H:%M:%S'));
+        s1 = str(psutil.cpu_count());
+        s2 = str(psutil.cpu_percent(interval=1));
+       	s3 = str(round(psutil.virtual_memory().total/(2**30), 2));
+        s4 = str(round(psutil.virtual_memory().used/(2**30), 2));
+
+        self.wfile.write(bytes(s0, "utf-8"));
```

Comment:
Web is a very high level form of programming and instead of writing a single stream of bytes, there is a much more structured way of passing data around the web (like between a webserver and a web page in POST requests)! This data format is called JSON and it looks like this.

```json
{
  "title": "some fruits!",
  "cost": 5,
  "fresh": false,
  "fruits":  [
    "apples",
    "bananas", 
    "oranges"
  ],
  "owner": {
    "name": "Enoch",
    "age": 43
  } 
}
```

Notice there are things like arrays, numbers, booleans. So it's very structured, the best part is that javascript is very good at parsing JSON! So from a webserver side of things, the best way to respond to a POST request is in JSON format.

Also since JSON is so popular, there's already a python library to format any data we have into JSON format. It's just called `json` and it can be imported with `import json`. Here's the official documentation page https://docs.python.org/3/library/json.html.